ts/features/bonus/bpd/components/bpdCardComponent/BpdCardComponent.tsx

```diff
@@ -172,7 +172,9 @@ const statusClosedHandler = (props: Props): GraphicalState => {
       totalAmount.transactionNumber < period.minTransactionNumber &&
       !isInGracePeriod
         ? ["0", "00"]
-        : formatNumberAmount(props.totalAmount.totalCashback).split(","),
+        : formatNumberAmount(props.totalAmount.totalCashback).split(
```

Comment:
I not sure about how this amount should be formatted.
Anyway, now this happens when the language is Italian (English is ok)

(amount is 1500 for test)
<img src="https://user-images.githubusercontent.com/822471/99103240-94d73a80-25df-11eb-9029-26c2c2e23cb0.png" height="600"/>

If this image represents a bug I suggest another approach by using this function

```typescript
// return an amount splitted in 2 parts:
// 1 - contains the integer part
// 2 - contains the decimal part
const getAmountSpliced = (amount: number): [string, string] => {
  const amountLocalizedSplitted = [
    ...formatNumberAmount(amount).split(
      I18n.t("global.localization.decimalSeparator")
    )
  ];
  return [
    take(amountLocalizedSplitted.length - 1, amountLocalizedSplitted).join(),
    last(amountLocalizedSplitted).getOrElse("00")
  ];
};
```
this is what you get using that function
<img src="https://user-images.githubusercontent.com/822471/99103408-ccde7d80-25df-11eb-8061-682f14cf9808.png" height="600"/>